packages/sharing-smart-contract/contracts/ProtectedDataSharing.sol

```diff
@@ -351,7 +371,8 @@ contract ProtectedDataSharing is ERC721Burnable, ERC721Receiver, ManageOrders, A
     function buyProtectedData(
         uint256 _collectionIdFrom,
         address _protectedData,
-        uint256 _collectionIdTo
+        uint256 _collectionIdTo,
+        address _appAddress
```

Comment:
why `_appAddress`? this param does not go well with the method name
or should we rename the method `buyProtectedDataForCollection`

It seems that the `buyProtectedData` (outside a collection is missing)

Reply:
the buyProtectedData (outside a collection is missing) is under this function. 
yeah i agree u changed the name

Reply:
Commit: 075890c6030300739b3e0cb49a153c47d3da3910